.changeset/dull-words-work.md

```diff
@@ -0,0 +1,11 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+---
+"section": tree
+---
+
+Enables Revertible objects to be cloned using `RevertibleAlpha.clone()`.
```

Comment:
```suggestion
Enables Revertible objects to be cloned using `RevertibleAlpha.clone()`
```

Reply:
https://github.com/microsoft/FluidFramework/pull/23174

This wasn't applied due to an auto-merge. Create a follow up PR for a fix.